src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -277,6 +281,13 @@ public function viewAction(int $orderId, Request $request): Response
         $addProductToOrderForm = $this->createForm(AddProductToOrderType::class);
         $updateOrderProductForm = $this->createForm(UpdateProductInOrderType::class);
 
+        $invoiceManagementIsEnabled = $this->getQueryBus()->handle(
+            new GetConfiguration(
+                Configuration::INVOICE_MANAGEMENT,
+                $orderForViewing->getShopId()
+            )
+        );
```

Comment:
@matks wdyt about this one? Current `ConfigurationInterface` is just broken, so I've implemented new query to get configuration values. I have a few thoughts left to discuss:

1. Does it make sense?
2. Should we keep `mixed` value as return type?
    If not, I can see at least 2 options here:
    1. We could ask query for specific type, e.g. `new GetConfiguration('KEY', null, null, ConfigType::BOOL)`, so value conversion will happen inside query handler.
    2. Or we could return DTO instead of `mixed`, e.g. `ConfigValue` as we could use it as `$configValue->asString()` or `$configValue->asBool()`, wdyt? 
3. It's meant for single configuration value, if we need to get multi language config, then we could add `GetMultilangConfiguration` query, would it make sense? Or we could add `$langId` to query, but that would make return value more complex.

Reply:
That is true, but on the other hand, we have both use case oriented queries and generic queries.

I was thinking about adding `GetInvoiceManagementIsEnabled` query, but not sure that's a good idea either. :/

How would you suggest going around it?

Reply:
Why dont we retrieve the data through `GetOrderForViewing` ?

Reply:
@sarjon Got a confirmation from Pablo: CQRS is a business-oriented interface, not a technical interface
so if `invoiceManagementIsEnabled` is required element for business in order to edit an order, then it must be returned by the query

Reply:
Ok, fixed!